core/timeline_api.php

```diff
@@ -101,11 +103,22 @@ function timeline_events( $p_start_time, $p_end_time, $p_max_events, $p_filter =
 						break;
 				}
 				break;
+			case FILE_ADDED:
+			case FILE_DELETED:
+				$t_event = new IssueAttachmentTimelineEvent(
+					$t_timestamp,
+					$t_user_id,
+					$t_issue_id,
+					$t_history_event['old_value'],
+					$t_type == FILE_ADDED
+				);
+ 				break;
 		}
 
 		# Do not include skipped events
 		if( $t_event != null && !$t_event->skip() ) {
 			$t_timeline_events[] = $t_event;
+			$t_previous_history_event = $t_history_event;
```

Comment:
Same as above, I don't see where this variable is used.

Reply:
Fixed.